frontend/src/components/postition-templates-list.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+export class PositionTemplatesList extends React.Component {
+    static propTypes = {
+        position_templates: PropTypes.arrayOf(
+            PropTypes.shape({
+                position_type: PropTypes.string,
+                offer_template: PropTypes.string
+            })
+        ).isRequired
+    };
+    render() {
+        const { position_templates } = this.props;
+        let templateList = <div>No Templates...</div>;
+        if (position_templates.length > 0) {
+            templateList = (
+                <ul>
+                    {position_templates.map(template => (
+                        <li key={template.offer_template}>
+                            <font size="3" color="blue">
```

Comment:
It would be better to use a `span` with a `className` here so that it can be styled with CSS